src/sentry/static/sentry/app/components/organizations/timeRangeSelector/dateRange.jsx

```diff
@@ -39,6 +40,11 @@ const DateRange = styled(
        */
       useUtc: PropTypes.bool,
 
+      /**
+       * handle UTC checkbox change
+       */
+      handleUseUtc: PropTypes.func,
```

Comment:
Use `on*` when it's a callback prop, and `handle*` for the function/method that handles a callback.

Reply:
🙏